iothub/device/tests/DeviceClientTests.cs

```diff
@@ -190,7 +191,7 @@ public async Task DeviceClient_OnMethodCalled_MethodRequestHasEmptyBody()
                 return Task.FromResult(new MethodResponse(Encoding.UTF8.GetBytes("{\"name\":\"ABC\"}"), 200));
             }, "custom data").ConfigureAwait(false);
 
-            var methodRequestInternal = new MethodRequestInternal("TestMethodName", "4B810AFC-CF5B-4AE8-91EB-245F7C7751F9", new MemoryStream(Array.Empty<byte>()), CancellationToken.None);
+            var methodRequestInternal = new MethodRequestInternal("TestMethodName", "4B810AFC-CF5B-4AE8-91EB-245F7C7751F9", new MemoryStream(new byte[0]), CancellationToken.None);
```

Comment:
what's the return type here? replace var probably?

Reply:
The return type is obvious because we're calling "new MethodRequestInterval". When it is obvious, we prefer "var" as the variable type.